packages/libs/react-components/src/components/Icons/styles.ts

```diff
@@ -20,6 +20,10 @@ export const sizeVariant = {
 export const IconContainer = styled('span', {
   display: 'flex',
 
+  '> svg': {
+    width: '100%',
+  },
```

Comment:
I'm a bit confused by this. Can you elaborate on the issue?

Reply:
oh i see 👍

Reply:
so if the icon is initially at size small and you resize the grid or icon to large the icon doesn't resize to the larger size

this because the svg has a maxWidth but not a width (which resizes the icon)